hoomd/Integrator.cc

```diff
@@ -911,26 +919,26 @@ void Integrator::setCommunicator(std::shared_ptr<Communicator> comm)
 void Integrator::computeCallback(uint64_t timestep)
     {
     // pre-compute all active forces
-    std::vector< std::shared_ptr<ForceCompute> >::iterator force_compute;
-
-    for (force_compute = m_forces.begin(); force_compute != m_forces.end(); ++force_compute)
-        (*force_compute)->preCompute(timestep);
+    for (auto& force : m_forces)
+        {
+        force->preCompute(timestep);
```

Comment:
I'm a little unsure whether this method is used? I only found two instances of "preCompute" in the code base, the `virtual void` declaration and this loop. From the understanding I got from some very quick reading, it might be possible to delete this and the calling `computeCallback` code.

Reply:
@jglaser might know.
I assume it is the vestige of a removed code path.